k4e-agent.spec

```diff
@@ -25,9 +25,27 @@ The K4E agent communicates with the K4E control plane. It reports the status of
 systemctl enable --now podman.socket
 systemctl enable --now nftables.service
 
+%prep 
+tar fx %{SOURCE0}
+
+%build
+cd k4e-agent-%{VERSION}
+export CGO_ENABLED=0 
+export GOFLAGS="-mod=vendor -tags=containers_image_openpgp"
+go build -o ./bin/device-worker ./cmd/device-worker
```

Comment:
In case of ARM 64 `GOARCH=arm64` is not set; will it be built correctly if the only indication of the architecture is `ExclusiveArch`?

Reply:
Mainly, on copr will detect that it's ARM host, and it'll run in there. ;-)